### DISCUSSION TITLE
WindSpeed discrepancy between UI & JSON

### DISCUSSION THREAD

**Discussion by mcarroll316** [weather-gov/api#353] [Q&A]
I'm curious to know the calculation used to display the Surface Wind (mph) in the UI.

One example is as such:

The [UI](https://forecast.weather.gov/MapClick.php?lat=44.9829&lon=-93.2771&lg=english&&FcstType=digital) may show "11" as the Wind Speed (mph), but the corresponding value in the json is 4.1155 (unit = kts).  I have ensured I've accounted for time values and am looking at the correct corresponding key/values.

Converting the knots to mph would be 4.7, not the 11 I see on the site.  In comparing the "11" with some other weather sites, it would be appear that that would be most accurate.  So, I'm curious as what the json value truly represents and/or if it's perhaps inaccurate.

**JSON Source**
https://api.weather.gov/gridpoints/MTR/89,126

Michael

**Comment by StephenClouse** [comment]
The wind values in the JSON are in meters per second, not knots, as noted by the "uom" property. I think you will find with this information that all the numbers do actually agree with each other.

The "sourceUnit" property that is confusing you is a bug we are fixing in our next release.
